iothub_client/samples/pnp/common/readme.md

```diff
@@ -0,0 +1,17 @@
+# Common helper utilities for PnP Sample
+
+This directory contains sample code that your application should be able to take with little or no modification to speed implementing a PnP device application.
+
+Probably the easiest way to understand how they work is to review the [pnp_temperature_controller](../pnp_temperature_controller) sample, which uses them extensively.
+
+For reference the files are:
+
+* `pnp_device_client_helpers` header and .c file implement a function to help create a `IOTHUB_DEVICE_CLIENT_HANDLE`.  The `IOTHUB_DEVICE_CLIENT_HANDLE` is an existing IoTHub Device SDK that can be used for device <=> IoTHub communication.  (There are many samples demonstrating its non-PnP uses in the [samples parent directory](../..).)
```

Comment:
The statement in the parenthesis is a complete sentence, not a random fragment, so I think there should be a period terminating it.  I also think the parenthesis makes sense because this isn't critical to understanding this sample.  So leaving as-is.